<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>atpInOrOut</class>
 <widget class="QDialog" name="atpInOrOut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_greeting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>19</y>
     <width>791</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Latin Modern Mono Light</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Hello</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>79</y>
     <width>791</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Latin Modern Mono Light</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>nume prenume</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>731</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Latin Modern Mono Light</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>731</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Latin Modern Mono Light</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
